src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ImageTests/Image_UseTargetSizeLate.xaml

```diff
@@ -19,7 +19,7 @@
 			   Width="100"
 			   Height="100"
 			   Stretch="UniformToFill"
-			   Source="https://lh5.ggpht.com/lxBMauupBiLIpgOgu5apeiX_YStXeHRLK1oneS4NfwwNt7fGDKMP0KpQIMwfjfL9GdHRVEavmg7gOrj5RYC4qwrjh3Y0jCWFDj83jzg" />
```

Comment:
Same here, needs to be delayed. Using github assets would be fine, though.

Reply:
Ok, I can use github, just I will probably need to push the images separately, or they won't be on master yet 😅

Reply:
If you don't rebase, the hash should be good to go :)

Reply:
@jeromelaban unfortunately the GitHub hosting is not an option, as it won't display on WASM this way (also because of the CORS issue). Do we have some server where the images could be put?